its/ruling/src/test/resources/expected/python-S3827.json

```diff
@@ -2,11 +2,6 @@
 'project:S1716.py':[
 4,
 ],
-'project:buildbot-0.8.6p1/buildbot/changes/bonsaipoller.py':[
-56,
```

Comment:
What's the change which fixed those FPs?

Reply:
The fix is here c25d408f92f56ac903d91e8cafa9413d2fe423d0